lib/bolt/plugin/terraform.rb

```diff
@@ -111,8 +111,10 @@ def extract_resources(state)
             resource_set['instances'].map do |resource|
               instance_name = prefix
               instance_name += ".#{resource['index_key']}" if resource['index_key']
-
-              [instance_name, resource['attributes']]
+              # When using `terraform state pull` with terraform >= 0.12 version 3 statefiles
+              # Will be converted to version 4. When converted attributes is converted to attributes_flat
+              attributes = resource['attributes'] || resource['attributes_flat']
+              [instance_name, attributes]
```

Comment:
Snippet from version 3 statefile generated with terraform 0.11:
```
            "resources": {
                "aws_instance.xenial.0": {
                    "type": "aws_instance",
                    "depends_on": [],
                    "primary": {
                        "id": "i-08594d1edf25dca11",
                        "attributes": {
```
Snippet from result of `terraform state pull` using terraform 0.12:
```
  "resources": [
    {
      "mode": "managed",
      "type": "aws_instance",
      "name": "xenial",
      "each": "list",
      "provider": "provider.aws",
      "instances": [
        {
          "index_key": 0,
          "schema_version": 1,
          "attributes_flat": {
            "ami": "ami-076e276d85f524150",
```